lib/modules/dosomething/dosomething_taxonomy/dosomething_taxonomy.module

```diff
@@ -117,14 +117,7 @@ function dosomething_taxonomy_preprocess_taxonomy_term(&$vars) {
  * @return array
  */
 function dosomething_taxonomy_get_campaigns($tid) {
-  $nids = array();
-  $view = views_get_view('campaigns_by_term');
-  $view->set_arguments(array($tid));
-  $view->execute();
-  foreach ($view->result as $record) {
-    $nids[] = $record->nid;
-  }
-  return $nids;
+  return taxonomy_select_nodes($tid, FALSE);
```

Comment:
Will this return non-campaign nodes assigned to that taxonomy? Is that a problem if so?